spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -344,10 +344,6 @@
         generate_saml_response(user)
       end
 
-      it 'stores the SAML Request original_url in the session' do
-        expect(session[:saml_request_url]).to eq(request.original_url)
-      end
```

Comment:
We need this test. `SignUp::CompletionsController` depends on this session var: https://github.com/18F/identity-idp/blob/master/app/controllers/sign_up/completions_controller.rb#L21